tests/linters_output/gitleaks.json

```diff
@@ -0,0 +1,16 @@
+{
+            "line": "-----BEGIN PRIVATE KEY-----",
+            "lineNumber": 59,
+            "offender": "-----BEGIN PRIVATE KEY-----",
+            "commit": "111111111111111111111000000000",
+            "repo": ".",
+            "repoURL": "",
+            "leakURL": "",
+            "rule": "Asymmetric Private Key",
+            "commitMessage": "any commit message \n",
+            "author": "bob s",
+            "email": "bob@example.com",
+            "file": "relative/path/to/output",
+            "date": "2020-04-14T15:17:53-07:00",
+            "tags": "key, AsymmetricPrivateKey"
+    }
```

Comment:
What does the gitleaks output look like if it finds more than one violation?  It would probably be a list instead containing dicts, right?

Also - Bonus points if you include a entropy based finding as a second result. (One regex, one entropy).

Reply:
Yes it will be a list. Ok I will look into it an in that case I have to modify my test caes as well.